lib/tasks/add_navigation_links.rake

```diff
@@ -1,25 +1,69 @@
 # rubocop:disable Metrics/BlockLength
 namespace :navigation_links do
+  reading_icon = File.read(Rails.root.join("app/assets/images/twemoji/drawer.svg")).freeze
+  contact_icon = File.read(Rails.root.join("app/assets/images/twemoji/contact.svg")).freeze
+  thumb_up_icon = File.read(Rails.root.join("app/assets/images/twemoji/thumb-up.svg")).freeze
+  smart_icon = File.read(Rails.root.join("app/assets/images/twemoji/smart.svg")).freeze
+  look_icon = File.read(Rails.root.join("app/assets/images/twemoji/look.svg")).freeze
+  listing_icon = File.read(Rails.root.join("app/assets/images/twemoji/listing.svg")).freeze
+  mic_icon = File.read(Rails.root.join("app/assets/images/twemoji/mic.svg")).freeze
+  camera_icon = File.read(Rails.root.join("app/assets/images/twemoji/camera.svg")).freeze
+  tag_icon = File.read(Rails.root.join("app/assets/images/twemoji/tag.svg")).freeze
+  bulb_icon = File.read(Rails.root.join("app/assets/images/twemoji/bulb.svg")).freeze
+  shopping_icon = File.read(Rails.root.join("app/assets/images/twemoji/shopping.svg")).freeze
+  heart_icon = File.read(Rails.root.join("app/assets/images/twemoji/heart.svg")).freeze
+  rainbowdev = File.read(Rails.root.join("app/assets/images/rainbowdev.svg")).freeze
+
+  desc "Create navigation links for new forem"
+  task create: :environment do
+    NavigationLink.where(url: "/readinglist").first_or_create(
+      name: "Reading List",
+      url: URL.url("readinglist"),
+      icon: reading_icon,
+      display_only_when_signed_in: true,
+      position: 0,
+      section: :default,
+    )
+    NavigationLink.where(url: "/contact").first_or_create(
+      name: "Contact",
+      url: URL.url("contact"),
+      icon: contact_icon,
+      display_only_when_signed_in: false,
+      position: 1,
+      section: :default,
+    )
+    NavigationLink.where(url: "/code-of-conduct").first_or_create(
+      name: "Code of Conduct",
+      url: URL.url("code-of-conduct"),
+      icon: thumb_up_icon,
+      display_only_when_signed_in: false,
+      position: 0,
+      section: :other,
+    )
+    NavigationLink.where(url: "/privacy").first_or_create(
+      name: "Privacy Policy",
+      url: URL.url("privacy"),
+      icon: smart_icon,
+      display_only_when_signed_in: false,
+      position: 1,
```

Comment:
I'll look since it's catching my attention - but should the position values be unique or are they used for priority (and we'll get the links back in some predetermined order and group by position)? 

I see now - there's a scope [`:ordered`](https://github.com/forem/forem/blob/main/app/models/navigation_link.rb#L14) that sorts by position, then by name within equal positions, in ascending (from 0 up, then alphabetical) order. The [controller](https://github.com/forem/forem/blob/main/app/controllers/admin/navigation_links_controller.rb#L8-L12) uses that to sort. 

All of my fears and doubts are soothed, and all is well with the world.

edit: Actually - now that I skimmed the RFC, I see the `section` attribute here is also important, the order for the static links is divided between "default" and "other" sections, and the sort indexes are relative to those sections.